bot/exts/evergreen/status_codes.py

```diff
@@ -1,10 +1,11 @@
-from http import HTTPStatus
-
 import discord
 from discord.ext import commands
 
 HTTP_DOG_URL = "https://httpstatusdogs.com/img/{code}.jpg"
 HTTP_CAT_URL = "https://http.cat/{code}.jpg"
+STATUS_TEMPLATE = '**Status: {code}**'
+ERR_404 = 'Unable to find status Floof for {code}.'
+ERR_UNKNOWN = 'Error attempting to retrieve status Floof for {code}.'
```

Comment:
These should be double quotes too
```suggestion
STATUS_TEMPLATE = "**Status: {code}**"
ERR_404 = "Unable to find status Floof for {code}."
ERR_UNKNOWN = "Error attempting to retrieve status Floof for {code}."
```